test/MusicStore.Spa.Test/ShoppingCartTest.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Http.Core.Collections;
+using Xunit;
+
+namespace MusicStore.Models
```

Comment:
Yes. it seemd strange to me. but the class that I'm testing is uder MusicStore.Models. And I think it should be fine for now because the test project has the reference to MusicStore.Spa only, so there will be no name collide. But I think we should clean up MusicStore and MusicStore.Spa samples for the new guildeline at some point, and I'd like to update this test at that time.